playbooks/nfsserver.yml

```
@@ -0,0 +1,18 @@
+---
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- name: configure nfsserver
+  hosts: lib_fs_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/nfsserver/{{ runtime_env | default('staging') }}.yml
+  roles:
+    - role: roles/nfsserver
```

Related to the comments on the `meta/main.yml` below, this is where you'd add the deploy user role: ```suggestion
    - role: roles/deploy_user
    - role: roles/nfsserver
```